Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
Wait, I mean to put `lint: venv`, not `lint: install-hooks` (I was using `install-hooks` as an example of another task that depends on `venv`).

Does it work with `venv` used instead?